src/platform/generic/_locales/bg/messages.json

```diff
@@ -0,0 +1,170 @@
+        "description": "Notification text when saving a password failed"
+    },
+    "UpdatedLoginDetectedTitle"  : {
+        "message"    : "Обновена регистрация",
+        "description": "Notification title when an updated login is detected"
+    },
+    "UpdatedLoginDetectedText"   : {
+        "message"     : "Щракнете тук за обновяване на парола за \"$USER$\"",
+        "description" : "Notification text when an updated login is detected",
+        "placeholders": {
+            "user": {
+                "content": "$1",
+                "example": "username"
+            }
+        }
+    },
+    "PasswordUpdatedTitle"       : {
+        "message"    : "Обновена парола",
+        "description": "Notification title when a password was updated"
+    },
+    "PasswordUpdatedText"        : {
+        "message"    : "Паролата е обновена успешно",
+        "description": "Notification text when a password was updated"
+    },
+    "UpdatePasswordFailedTitle"  : {
+        "message"    : "Грешка при обновяване на парола",
+        "description": "Notification title when updating a password failed"
+    },
+    "UpdatePasswordFailedText"   : {
+        "message"    : "Паролата не е обновена поради грешка",
+        "description": "Notification text when updating a password failed"
+    },
+    "ApiLogin200"   : {
+        "message"    : "Недействителен адрес на URL или изключен ППИ",
+        "description": "Error text when an api request could not be decoded"
+    },
+    "ApiLogin401"   : {
+        "message"    : "Грешка при удостоверяване",
+        "description": "Error text when an api request got a 401 Unauthorized response"
+    },
+    "ApiLogin403"   : {
+        "message"    : "Достъпът отказан",
+        "description": "Error text when an api request got a 403 Forbidden response"
+    },
+    "ApiLogin404"   : {
+        "message"    : "Недействителен адрес на URL",
+        "description": "Error text when an api request got a 404 Not Found response"
+    },
+    "ApiLogin500"   : {
+        "message"    : "Сървърът се срина при заявката",
+        "description": "Error text when an api request got a 50x Internal Server Error response"
+    },
+    "ApiLoginGeneric"   : {
+        "message"    : "Http Error $STATUS$: $MESSAGE$",
+        "description": "Generic error message for failed requests",
+        "placeholders": {
+            "status": {
+                "content": "$1",
+                "example": "HTTP status code"
+            },
+            "message": {
+                "content": "$2",
+                "example": "HTTP status text"
+            }
+        }
+    },
+    "SettingsInvalidUrl"   : {
+        "message"    : "Адресът $URL$ изглежда е недействителен.\n\nАко приложението се намира на\nhttps://example.com/index.php/apps/passwords\nвъведете\nhttps://localhost\n\nАко приложението се намира на\nhttps://example.com/nextcloud/index.php/apps/passwords\nвъведете\nhttps://localhost/nextcloud,
```

Comment:
there seems to be a missing " before the comma which breaks the JSON. can you fix it?

Reply:
```suggestion
        "message"    : "Адресът $URL$ изглежда е недействителен.\n\nАко приложението се намира на\nhttps://example.com/index.php/apps/passwords\nвъведете\nhttps://localhost\n\nАко приложението се намира на\nhttps://example.com/nextcloud/index.php/apps/passwords\nвъведете\nhttps://localhost/nextcloud",
```